initial_bootstrap/Dockerfile

```diff
@@ -32,11 +32,7 @@ RUN cat /proc/cpuinfo && cat /proc/meminfo && df -h
 RUN wget https://gitweb.gentoo.org/repo/proj/prefix.git/plain/scripts/bootstrap-prefix.sh && chmod +x bootstrap-prefix.sh
 ENV EPREFIX /tmp/gentoo
 
-# Needed for 32b
-ENV CHOST i686-pc-linux-gnu
```

Comment:
Well, running bootstrap-prefix.sh using that 'linux32' really should be sufficient here.

Reply:
I tried that (for a while, with help from the IRC channel) and some packages managed to ignore it somehow and I only found on runtime after building the prefix. By what I remember something still generated 64b binaries/shared libraries. The only way it worked was with the CHOST flag. Can you please bring it back?

I'd give it a second try without it but im currently busy to give it a well thru test.

Reply:
Hmm... bootstrap-prefix.sh really should identify CHOST on its own, based on 'uname -m' output:
$ uname -m
x86_64
$ linux32 uname -m
i686

Reply:
I know. And that's what others said in the IRC channel. But my experience after trying-to build it, it would fail, or after building it, libraries would fail with incompatible 32-64b problems.

Given it took me a week to get back to you again, I'll just merge it, if it doesn't work as expected, I'll revert it.